### PR TITLE
Locator branch

### PR DESCRIPTION
**Pull request by rpallavisharma**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description

Added locator information in the locator strategies page
### Motivation and Context

This section was missing. So added
### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/config.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ theme = ["docsy"]
 
 # Will give values to .Lastmod etc.
 enableGitInfo = true
-#ignoreErrors = ["error-remote-getjson"] 
+ignoreErrors = ["error-remote-getjson"]
```
Comment (harsha509) [inline website_and_docs/config.toml]: i think this should be commented out!

File: website_and_docs/content/documentation/webdriver/elements/information.en.md [hunk 2]
```diff
@@ -47,7 +47,11 @@ driver.Url = "https://www.selenium.dev/selenium/web/inputs.html";
 Boolean is_email_visible = driver.FindElement(By.Name("email_input")).Displayed;
 {{< /tab >}}
 {{< tab header="Ruby" >}}
-# Help us with a PR for code sample
+# Navigate to the url
+driver.get("https://www.selenium.dev/selenium/web/inputs.html");
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/elements/information.en.md]: not sure if it is mandatory. Do we need semi-colon in rb ?

File: website_and_docs/config.toml (hunk 1 above)
Comment (rpallavisharma) [reply website_and_docs/config.toml]: hugo server doesnt work if i comment this out Harsha, please suggest.

Comment (diemol) [reply website_and_docs/config.toml]: What error do you get?

Comment (titusfortner) [reply website_and_docs/config.toml]: This error happens when Github API limit is exceeded.

**Comment by rpallavisharma** [comment]
I have made changes please check

**Comment by diemol** [comment]
Was this intentionally closed, @pallavigitwork? Should we still check this PR?

**Comment by rpallavisharma** [comment]
yes. i was unable to run hugo server. so closed this. https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1337 is the latest PR now. i hope i have done this time correctly. but please check and let me know. @harsha509

**Pull request closed without merging**